sql/src/main/java/org/apache/druid/sql/calcite/schema/InformationSchema.java

```diff
@@ -131,6 +139,14 @@ public RelDataType build()
       .add("COLLATION_NAME", SqlTypeName.VARCHAR, true)
       .add("JDBC_TYPE", SqlTypeName.BIGINT)
       .build();
+  private static final RelDataType ROUTINES_SIGNATURE = new RowTypeBuilder()
+      .add("ROUTINE_CATALOG", SqlTypeName.VARCHAR)
+      .add("ROUTINE_SCHEMA", SqlTypeName.VARCHAR)
+      .add("ROUTINE_NAME", SqlTypeName.VARCHAR)
+      .add("ROUTINE_TYPE", SqlTypeName.VARCHAR)
+      .add("IS_AGGREGATOR", SqlTypeName.VARCHAR)
+      .add("SIGNATURES", SqlTypeName.VARCHAR)
```

Comment:
seems that this column can be nullable based on logic below.

Reply:
thanks, added.